src/M6Web/Bundle/LogBridgeBundle/DependencyInjection/M6WebLogBridgeExtension.php

```diff
@@ -72,4 +75,41 @@ protected function loadListener(ContainerBuilder $container)
         $container->setDefinition($serviceName, $definition);
     }
 
+
+    /**
+     * loadDefaultLogger
+     *
+     * @throws \Exception Logger service must be implement "Psr\Log\LoggerInterface"
+     * @throws \Exception Logger channel must be a string
+     *
+     * @param ContainerBuilder $container Container
+     */
+    protected function loadDefaultLogger(ContainerBuilder $container)
```

Comment:
I keep convinced that this is not a load**Default**Logger method, but a loadLogger one.